index.html

```diff
@@ -4709,6 +4708,18 @@ <h2 id="AudioBufferSourceNode">
           This node has no <a>tail-time</a> reference.
         </p>
         <p>
+          A <dfn>playhead position</dfn> for an <a>AudioBufferSourceNode</a> is
```

Comment:
This seems to be in a weird place.  It seems that it's really only relevant to the playback section and might be better to move the paragraph there.